src/Database/Migrations/2021-07-04-041948_CreateSettingsTable.php

```diff
@@ -2,10 +2,22 @@
 
 namespace CodeIgniter\Settings\Database\Migrations;
 
+use CodeIgniter\Config\BaseConfig;
```

Comment:
It is okay to require the Config to be ours (or an extension thereof):
```suggestion
use CodeIgniter\Settings\Config\Settings;
```